configs/train_config.yaml

```diff
@@ -8,14 +8,42 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
+### Data Pre-processing parameters
+## Generate intervals files for training, validation
+interval_size: 50000
+val_chrom: "None"
+holdout_chrom: "None"
+# Chromosome sizes to be used
+# You can also provide a file path
+sizes_file: "hg19"
+# Ratio of non-peak to peak intervals in training
+# non-peak samples = nonpeak * peak samples
+nonpeak: "None"
+
+## Input files
+noisybw: "None"
+cleanbw: "None"
+layersbw: "None"
+# This is the number of lines of bigiwig files to read at once, because
+# the entire file may not fit in memory if read at once.
+read_buffer: 1000
 
 # Experiment args
 out_home: './'
 label: 'trained_models'
+# If you wish to train a model to just denoise coverage track
```

Comment:
Similarly, this and other commented lines in this file look like they can be moved to help options of cmd_args.py

Reply:
done